hedera-node/docker/.dockerignore

```diff
@@ -10,6 +10,7 @@ docs/
 hapi-proto/target/
 hedera-node/data/lib/
 hedera-node/data/recordstreams/
+hedera-node/data/blockstreams/
```

Comment:
Both of these, should they be blockstreams or blockStreams (recordstreams vs. recordStreams)

Reply:
Also fixed